protocol-designer/src/components/organisms/OverlayModal/index.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  ALIGN_CENTER,
+  COLORS,
+  DIRECTION_COLUMN,
+  DIRECTION_ROW,
+  Flex,
+  JUSTIFY_CENTER,
+  JUSTIFY_SPACE_BETWEEN,
+  Overlay,
+  PrimaryButton,
+  SecondaryButton,
+  SPACING,
+  StyledText,
+  TEXT_ALIGN_CENTER,
+} from '@opentrons/components'
+
+export interface OverlayModalProps {
+  header: string
+  subText?: string
+  primaryButtonText: string
+  secondaryButtonText: string
+  onSecondaryButtonClick: () => void
+  onPrimaryButtonClick: () => void
+}
```

Comment:
nit: since subText is optional, can it be listed last? 

```suggestion
export interface OverlayModalProps {
  header: string
  primaryButtonText: string
  secondaryButtonText: string
  onSecondaryButtonClick: () => void
  onPrimaryButtonClick: () => void
    subText?: string
}
```